packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-matrix.stories.ts

```diff
@@ -64,6 +86,77 @@ export const richTextEditorSizing: StoryFn = createStory(html`
     ])}
 `);
 
+const styledComponent = (): ViewTemplate => html`
+<div style="padding: 20px">
+    <${richTextEditorTag}>
+        <${buttonTag} slot="footer-actions" appearance="ghost">Cancel</${buttonTag}>
+        <${buttonTag} slot="footer-actions" appearance="outline">Ok</${buttonTag}>
+    </${richTextEditorTag}>
+</div>
+`;
+
+const mobileWidthComponent = (): ViewTemplate => html`
+<div style="padding: 20px; width: 300px;">
+    <${richTextEditorTag}>
+        <${buttonTag} slot="footer-actions" appearance="ghost">Cancel</${buttonTag}>
+        <${buttonTag} slot="footer-actions" appearance="outline">Ok</${buttonTag}>
+    </${richTextEditorTag}>
+</div>
+`;
+
+export const editorWhenMarkdownValueSetInLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    createMatrix(styledComponent),
+    lightThemeWhiteBackground
+);
+
+editorWhenMarkdownValueSetInLightThemeWhiteBackground.play = playFunction;
+
+export const editorWhenMarkdownValueSetInColorThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
+    createMatrix(styledComponent),
+    colorThemeDarkGreenBackground
+);
+
+editorWhenMarkdownValueSetInColorThemeDarkGreenBackground.play = playFunction;
+
+export const editorWhenMarkdownValueSetInDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
+    createMatrix(styledComponent),
+    darkThemeBlackBackground
+);
+
+editorWhenMarkdownValueSetInDarkThemeBlackBackground.play = playFunction;
+
+export const plainTextContentInMobileWidth: StoryFn = createFixedThemeStory(
+    createMatrix(mobileWidthComponent),
+    lightThemeWhiteBackground
+);
+
+plainTextContentInMobileWidth.play = (): void => {
+    document.querySelector('nimble-rich-text-editor')!.setMarkdown(loremIpsum);
+};
+
+export const multipleSubPointsContentInMobileWidth: StoryFn = createFixedThemeStory(
+    createMatrix(mobileWidthComponent),
+    lightThemeWhiteBackground
+);
+
+multipleSubPointsContentInMobileWidth.play = (): void => {
+    document
+        .querySelector('nimble-rich-text-editor')!
+        .setMarkdown(
+            '1. Sub point 1\n   1. Sub point 2\n       1. Sub point 3\n          1. Sub point 4\n             1. Sub point 5\n                 1. Sub point 6\n                    1. Sub point 7\n'
```

Comment:
Can you format this string as a template literal so we can see how the indentations line up?
```
`1. Sub point 1
    1. Sub point 2
        1. Sub point 3
        ...
`
```

Reply:
Updated